app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -0,0 +1,190 @@
+
+  var selectedTabIndex by remember { mutableIntStateOf(0) }
+
+  Column(modifier = Modifier.fillMaxSize()) {
+    Spacer(modifier = Modifier.height(50.dp))
+    Row(
+        modifier = Modifier.fillMaxWidth().padding(start = 32.dp),
+        verticalAlignment = Alignment.CenterVertically) {
+          Image(
+              painter = painterResource(id = R.drawable.event_logo),
+              contentDescription = "Event Radar Logo",
+              modifier = Modifier.size(width = 184.35.dp, height = 46.99.dp))
+        }
+    Spacer(modifier = Modifier.height(16.dp))
+    TabRow(
+        selectedTabIndex = selectedTabIndex,
+        modifier = Modifier.fillMaxWidth().height(96.dp).padding(32.dp),
+        contentColor = MaterialTheme.colorScheme.primary) {
+          Tab(
+              selected = selectedTabIndex == 0,
+              onClick = { selectedTabIndex = 0 },
+          ) {
+            Text(
+                text = "Browse",
+                style =
+                    TextStyle(
+                        fontSize = 19.sp,
+                        lineHeight = 17.sp,
+                        fontFamily = FontFamily(Font(R.font.roboto)),
+                        fontWeight = FontWeight(500),
+                        color = MaterialTheme.colorScheme.onPrimaryContainer,
+                        textAlign = TextAlign.Center,
+                        letterSpacing = 0.25.sp,
+                    ))
+          }
+          Tab(selected = selectedTabIndex == 1, onClick = { selectedTabIndex = 1 }) {
+            Text(
+                text = "Upcoming",
+                style =
+                    TextStyle(
+                        fontSize = 19.sp,
+                        lineHeight = 17.sp,
+                        fontFamily = FontFamily(Font(R.font.roboto)),
+                        fontWeight = FontWeight(500),
+                        color = MaterialTheme.colorScheme.onPrimaryContainer,
+                        textAlign = TextAlign.Center,
+                        letterSpacing = 0.25.sp,
+                    ))
+          }
+        }
+    Spacer(modifier = Modifier.height(8.dp))
+    if (selectedTabIndex == 0) {
+      EventList(mockEvents)
+    } else {
+      // "Upcoming" tab content
+      Box(contentAlignment = Alignment.Center, modifier = Modifier.fillMaxSize()) {
+        Text(
+            text = "Upcoming events feature not implemented yet",
+            style =
+                TextStyle(
+                    fontFamily = FontFamily(Font(R.font.roboto)),
+                    fontSize = 18.sp,
+                    fontWeight = FontWeight(500)),
+            textAlign = TextAlign.Center)
+      }
+    }
+  }
+}
+
+@Composable
+fun EventList(events: List<Event>) {
+  LazyColumn { items(events) { event -> EventCard(event) } }
+}
+
+@Composable
+fun EventCard(event: Event) {
+  Card(
+      modifier = Modifier.padding(horizontal = 16.dp, vertical = 8.dp).height(IntrinsicSize.Min),
```

Comment:
Here I would use a constraint layout to fix the left and right of the cards to the left and right, respectively, of the `TabRow`. 
```suggestion
      modifier = Modifier.padding(horizontal = 32.dp, vertical = 8.dp).height(IntrinsicSize.Min),
```
A temporary quick fix could be simply having the same horizontal padding as the `TabRow`